listener.js

```diff
@@ -179,11 +185,35 @@ server.post("/", (payload, res) => {
             // CHANGE DEVICE BRIGHTNESS
             case "brightness":
                 if (device.name == target.device) {
-                    let brightness = await cli_exec("curl -X POST http://" + device.ipaddr + ":8080/brightness?value=" + target.value, "device_command");
+                    var ipaddr = '';
+                    if (config.manual_ip) {
+                        ipaddr = device.ipaddr;
+                    }
+                    else {
+                        // Look for WiFi addresses since it's quick
+                        ipaddr = await cli_exec("ping -t 1 " + device.name, 'device_ipaddr');
+                        if (ipaddr == '') {
+                            // Look for tethered addresses and blanks. This takes a while
+                            ipaddr = await cli_exec("grep -A1 \"" + device.name.replace('+', '.*') + "\" /var/db/dhcpd_leases", 'device_ipaddr');
+                        }
+                    }
+                    let brightness = await cli_exec("curl --connect-timeout 10 -m 10 -X POST http://" + ipaddr + ":8080/brightness?value=" + target.value, "device_command");
```

Comment:
```suggestion
                    const brightness = await cli_exec("curl --connect-timeout 10 -m 10 -X POST http://" + ipaddr + ":8080/brightness?value=" + target.value, "device_command");
```
Since brightness value isn't changed